src/components/MultiHopTrade/components/LimitOrder/hooks/useAllowanceApproval.tsx

```diff
@@ -95,7 +95,11 @@ export const useAllowanceApproval = ({
   })
 
   return {
-    isLoading: allowanceCryptoBaseUnitResult.isLoading || evmFeesResult.isLoading,
+    isLoading:
+      allowanceCryptoBaseUnitResult.isLoading ||
+      evmFeesResult.isLoading ||
+      allowanceCryptoBaseUnitResult.isRefetching ||
+      evmFeesResult.isRefetching,
```

Comment:
Allowance can and will refetch at interval, but we were not capturing this properly, meaning only initial loading state would trigger a skeleton